src/main/java/com/teenthofabud/wizard/nandifoods/wms/settings/unit/uom/converter/UOMEntityToDtoV2Converter.java

```diff
@@ -47,8 +52,11 @@ public UOMDtoV2 convert(UOMEntity source) {
                 .isPurchase(source.getIsPurchase())
                 .status(source.getStatus())
                 .measuredValues(measuredValues)
+                .linkedUOMs(linkedUOMs)
```

Comment:
Remove linkedUOMs field from DTO and adjust logic accordingly. Linked UOM should exist as a dedicated resource with loose coupling to UOMDto by not being a collection property of UOMDto of type UOMCrossLinkage and UOMSelfLinkage but rather highly cohesive with the UOMDTo's id as a property of UOMCrossLinkage and UOMSelfLinkage